core/conn/odbc/src/odbc/nsksrvrcore/ResStatisticsStatement.cpp

```diff
@@ -1265,7 +1265,7 @@ void ResStatisticsStatement::setStatistics(SRVR_STMT_HDL *pSrvrStmt, SQLSTATS_TY
 #define MAX_PERTABLE_STATS_DESC		30
 
 #define MAX_MASTERSTATS_ENTRY		31
-#define MAX_MEASSTATS_ENTRY			34
+#define MAX_MEASSTATS_ENTRY			26
 #define MAX_PERTABLE_ENTRY			14
```

Comment:
Yes. Changed it.  Missed out in ODBC code but changed it in Type2 JDBC driver. I think there is a need to share this code between all components SQL, JDBC and ODBC.